### PR TITLE
fix(CX-3703): fix errors in the contact the specialist form

### PR DESCRIPTION
**Pull request by dariakoko**
This PR resolves [CX-3703] 

### Description

https://github.com/artsy/eigen/assets/36167539/e77885d5-e3e8-4128-9f76-db8d2e42dbb3

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [ ] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [ ] I have added **tests**, or my changes don't require any.
- [ ] I added an **[app state migration]**, or my changes do not require one.
- [ ] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [ ] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- fix errors in the contact the specialist form -daria

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[CX-3703]: https://artsyproduct.atlassian.net/browse/CX-3703?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by MounirDhahri — DISMISSED** [review]
Nice! thanks Daria. 
Can we also have validation only on submit - currently it's on change I believe

**Comment by MounirDhahri** [comment]
Please feel free to merge afterward

**Comment by dariakoko** [comment]
@MounirDhahri I checked how we do it on a similar form - request a price estimate. There we validate onchange as well
I'd keep it as it is for now. What do you think?

**Comment by MounirDhahri** [comment]
I actually disagree with having it that way in the previous form as well. I think we probably missed it there. I feel it's not a good UX to immediately raise raise errors to users when they're typing on field 1 to tell them field 2 is missing unleass they touched field 2. We can validate onBlur maybe the touched fields 👀

**Comment by dariakoko** [comment]
@MounirDhahri Sure, if you think it's better
I just think that if we fix these types of issues abstracted from each other they will never reach consistency.
Would you like to jump on a quick call?

File: src/app/Scenes/SellWithArtsy/ConsignmentInquiry/ConsignmentInquiryScreen.tsx [hunk 1]
```diff
@@ -96,9 +96,11 @@ export const ConsignmentInquiryScreen: React.FC<InquiryScreenProps> = ({
     },
 
     onSubmit: async ({ email, name, phoneNumber, message }) => {
+      const formattedValue = name.trim()
```
Comment (lordkiz) [inline src/app/Scenes/SellWithArtsy/ConsignmentInquiry/ConsignmentInquiryScreen.tsx:99]: *Non-blocking*: The value is best named to reflect what it is: You might want to consider naming it `formattedNameValue` instead

**Review by lordkiz — DISMISSED** [review]
👍 👍 👍

**Review by MrSltun — DISMISSED** [review]

File: src/app/Scenes/SellWithArtsy/ConsignmentInquiry/ConsignmentInquiryForm.tsx [hunk 2]
```diff
@@ -61,16 +61,14 @@ export const ConsignmentInquiryForm: React.FC<{
   // multiline input and the send button at the same time, hence the need
   // for this function
   const showMessageInputFully = () => {
-    requestAnimationFrame(() => {
+    /*  requestAnimationFrame(() => {
       ScrollViewRef.current?.scrollToEnd()
-    })
+    }) */
```
[the remark this answers is not in the document]
Comment (dariakoko) [reply src/app/Scenes/SellWithArtsy/ConsignmentInquiry/ConsignmentInquiryForm.tsx:66]: i'll test on android, i think we should fix it

[the remark this answers is not in the document]
Comment (dariakoko) [reply src/app/Scenes/SellWithArtsy/ConsignmentInquiry/ConsignmentInquiryForm.tsx:66]: Takes long to test, I need to update my android env
Will come back to it later
For now, keeping this function in order not to introduce a new issue before the release

**Review by MounirDhahri — APPROVED** [review]

**Comment by MounirDhahri** [comment]
🙌🙌🙌

**Pull request merged**